### DISCUSSION TITLE
Is it possible to run under a subfolder over Apache proxy?

### DISCUSSION THREAD

**Discussion by gerroon** [advplyr/audiobookshelf#1524] [Q&A]
Hi

I want to run ABS under something like `https:/mydomain.com/audiobookshelf` The reverse proxy example for Apache seems to assume it will be running under its own subdomain.

thanks

**Comment by hobesman** [comment]
To my knowledge nobody has gotten it to work using a subfolder, but I also don't think it's been a priority for the devs.

I imagine if it's something you know how to code for, they would be receptive to any code contribution you can make.

**Reply by advplyr** [reply]
Correct, thanks. The issue for that is here https://github.com/advplyr/audiobookshelf/issues/385

**Reply by gerroon** [reply]
Thanks. I am not a programmer so I can't really help with that :(
